app/Http/Controllers/HomeController.php

```diff
@@ -46,7 +46,12 @@ public function profileReservation(){
         return view('users.profilereservation');
     }
 
+<<<<<<< HEAD
```

Comment:
@Tommmy1018 You included in your commit the conflict.

Reply:
What does that mean? I couldn't find any conflict in the file.